.github/workflows/cypress.yml.disabled

```diff
@@ -6,134 +6,42 @@ on:
   pull_request:
 
 env:
-  INSTANCE_TYPE: m5.large
   NODE_EXTRA_CA_CERTS: /etc/ssl/certs/ca-certificates.crt
   OPENSSL_CONF: /etc/ssl/openssl.cnf
+  CYPRESS_INSTALL_BINARY: 0
+  CYPRESS_CACHE_FOLDER: /home/runner/runner1/_work/next-build/next-build/.yarn/cypress/cypress.zip
```

Comment:
`runner1` is not guaranteed; there are usually 3 runners per ASG instance. This would need to be relative or otherwise variable.

Reply:
yeah good callout. this workflow is still disabled for the time being but i'll add a comment